src/js/touchevents.tsx

```diff
@@ -70,6 +72,15 @@ class TouchEventBoundary extends React.Component<TouchEventBoundaryProps> {
     maxComponentTreeSize: DEFAULT_MAX_COMPONENT_TREE_SIZE,
   };
 
+  public readonly name: string = 'TouchEventBoundary';
+
+  /**
+   * Registers the TouchEventBoundary as a Sentry Integration.
+   */
+  public componentDidMount(): void {
+    getCurrentHub().getClient()?.addIntegration?.(createIntegration(this.name));
+  }
```

Comment:
L: I believe this could be called multiple times, we just need to be sure that we don't add duplicates to the list.

Reply:
I'm aware of that, I didn't put any check here, as the function checks if the integration was installed.